tools/vagrant/vagrant-libvirt.md

```diff
@@ -51,9 +51,9 @@ To revert for your box to `qemu://system`, you can put something like the follow
 ```ruby
 Vagrant.configure("2") do |config|
   config.vm.provider :libvirt do |libvirt|
-    # Use QEMU session instead of system connection
-    libvirt.qemu_use_session = true
-
+    # Use QEMU system instead of session connection
+    libvirt.qemu_use_session = false
+end
```

Comment:
This was left without an end keyword on purpose. If we add the `end` it should be syntactically valid at least from Ruby POV.
```suggestion
    libvirt.qemu_use_session = false
  end
end
```

Reply:
I trust you on this :)